src/components/LibraryContainer.tsx

```diff
@@ -1,5 +1,4 @@
 /// <reference path="../../node_modules/@types/node/index.d.ts" />
-/// <reference path="../../node_modules/@types/whatwg-fetch/index.d.ts" />
```

Comment:
Thanks for the suggestion, @ramramps, do you have some examples of that? We can start looking into WebUI component for info if that's easier for you.